plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/SortFieldUtils.java

```diff
@@ -72,6 +73,26 @@ public static SortOrder parseSortFields(Schema schema, List<String> fields)
         return sortOrder;
     }
 
+    private static Set<Integer> collectBaseColumnFieldIds(Schema schema)
```

Comment:
Methdo name is misleading. 

```suggestion
private static Set<Integer> collectColumnFieldIds(Schema schema)
```

Reply:
Fixed